stories/Colors/Colors.jsx

```diff
@@ -198,6 +198,11 @@ export default () => (
     <Title as="h3" style={{ paddingTop: 0 }}>
       Black
     </Title>
+    <p>
```

Comment:
The new colors update on documentation is fine, but we need do update all the other colors too.
Notice that colors and his names is all deprecated. we use six colors level now (except the neutral).
We can do it together and show to Arthur, what do you think?